elasticai/creator/systemTests/models_definition.py

```diff
@@ -138,19 +128,27 @@ def crete_brevitas_model():
             in_features=15,
             out_features=1,
             bias=False,
-            weight_quant=bquant.BinaryWeights
+            weight_quant=bquant.BinaryWeights,
         ),
-        nn.Sigmoid()
+        nn.Sigmoid(),
     )
     return model
 
 
-weigth_layers = (QConv1d, QLinear, nn.BatchNorm1d, bnn.QuantConv1d, bnn.QuantLinear)
+weight_layers = (QConv1d, QLinear, nn.BatchNorm1d, bnn.QuantConv1d, bnn.QuantLinear)
 
 
 def define_weight(layers):
+    # TODO: In case we're using this function just to have deterministic weight value generation
+    #  we should reconsider our approach here
```

Comment:
do you have a different idea in mind?

Reply:
couldn't we just set the random seeds accordingly?